massgov/mayflower-patternlab
# DP-6181 [a11y] Add supplemental info to phone and email links for screen readers

## Description

When screen reader users browse a page _from link to link_ and hit contact information such as **phone numbers** and **email addresses**, they don't get what those links are for.  To support that, add their contact name as hidden text only read by screen readers to phone number and email links.

## Related Issue / Ticket

- [DP-6181 ]A11y use aria-label to provide additional text to screen reader user for phone and email links()

## Steps to Test


1. Go to pages with contact information such as Location, Decision, Event, Guide and etc.
2. Check **phone number** and **email links** in Inspect, Firebug or source code.
3. Find such links contain `span.visually-hidden` with their contact name, which is shown in `.ma__contact-group__name`. For phone numbers, the `span` includes value of `label`, which often shown as "Main", "Toll free" and such, if available.
```
<a class="ma__content-link ma__content-link--phone" href="tel:+14134994262">
<span class="visually-hidden">Call Office of Consumer Affairs & Business Regulation (OCABR), Main: at </span>
(413) 499-4262
</a>
...
<a class="ma__content-link" href="mailto:email@email.com">
<span class="visually-hidden">Email Office of Consumer Affairs & Business Regulation (OCABR) at </span>
email@email.com
</a>
```

## Screenshots
<img width="717" alt="dp-6181_after" src="https://user-images.githubusercontent.com/9633303/35534441-894dfaf4-050e-11e8-90c4-5c93fc1d97b9.png">

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
